### ISSUE TITLE
Is setting consistent-read supported by the API  ?

### ISSUE THREAD

**Issue by fabapp** [derjust/spring-data-dynamodb#83]
Hello,

is it somehow supported (planned) to allow setting ConsistentRead=true ?
(e.g.: http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_GetItem.html#API_GetItem_RequestSyntax)
I searched the sourceode but could't find anything that could solve this.

Thank you in advance!

Cheers,
Fabian

**Comment by gauravbrills** [comment]
I am also looking for an answer to the same ,looking at the code and setting DynamoDbConfig seems the mapper used in the api is not using the bean provided with CONSISTENT reads . Will dig through the code more to find a way to inject the same .Though any help will be great 
@derjust

**Comment by derjust** [comment]
Yes @gauravbrills is right you have to do it with those two things (configuration bean + `configRef`) - annoying and should be improved - but that's the way to do it right now.
